src/server/kiwix-serve.cpp

```diff
@@ -74,6 +74,7 @@ void usage()
             << "\t-V, --version\t\tprint software version" << std::endl
             << "\t-z, --nodatealiases\tcreate URL aliases for each content by removing the date" << std::endl
             << "\t-c, --customIndex\tadd path to custom index.html for welcome page" << std::endl
+            << "\t-L, --ipConnectionLimit\tLimit on the number of (concurrent) connections made to the server from the same IP address (default: infinite, recommended: >= 6)" << std::endl
```

Comment:
Unlike other options the description of this new option starts with an uppercase letter.

Reply:
This should be coherent, to me all comments should start with uppercase

Reply:
I have fixed it.